lib/object-definition.js

```diff
@@ -187,10 +187,21 @@ ObjectDefinition.prototype.defineProperty = function(property) {
       if (defined) definition.example = defined.example
       return defined
     }));
-
-  // If the property value is an object and has its own properties,
-  // make them available to the definition
-  } else if (property.properties) {
+  } else if (property.items?.oneOf || property.items?.anyOf){
+    var key = property.items.oneOf ? 'oneOf' : 'anyOf';
+    definition[key] = _.filter(_.map(property.items.oneOf || property.items.anyOf, (object) => {
```

Comment:
We may want to switch to ES6 conventions and remove Lodash at some point—not as part of this, but as something to do in the future.

Reply:
I hope we move to OpenAPI 3 and ditch this repository altogether 🗡️